src/components/DocumentScanner.tsx

```
@@ -0,0 +1,272 @@
+        doc_type: docConfig.doc_type,
+        doc_subtype: docConfig.doc_subtype,
+        doc_data: jsonData,
+        uploaded_at: new Date().toISOString(),
+        imported_from: "e-wallet",
+        doc_datatype: "Application/JSON"
+      }];
+
+      // Upload the document
+      await uploadUserDocuments(documentPayload);
+
+      // Refresh user data to update the UI
```

ESLint

[error] Replace `······` with `↹↹↹`

(prettier/prettier)